src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs

```diff
@@ -1078,11 +1079,64 @@ public void Select(int start, int length)
 
 		public void SelectAll() => SelectAllPartial();
 
-
 		partial void SelectPartial(int start, int length);
 
 		partial void SelectAllPartial();
 
+		/// <summary>
+		/// Copies content from the OS clipboard into the text control.
+		/// </summary>
+		public void PasteFromClipboard()
+		{
+			Dispatcher.RunAsync(CoreDispatcherPriority.High, async () =>
```

Comment:
I think this should fix the build error

```suggestion
			_ = Dispatcher.RunAsync(CoreDispatcherPriority.High, async () =>
```